test/e2e/linuxbridge_test.go

```diff
@@ -0,0 +1,106 @@
+package e2e
+
+import (
+	"context"
+	"io/ioutil"
+	"testing"
+	"time"
+
+	yaml "github.com/ghodss/yaml"
```

Comment:
is there a specific pattern you follow when grouping and sorting imports?

Reply:
I think we usually have first golang core, then third parties (puting k8s first), then app.
Let me fix it.

Also going to use k8s yaml parser sigs.k8s.io/yaml